### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by thomashaenig**
Signed-off-by: Thomas Hänig <thomas.haenig@akquinet.de>

### CODE REVIEW

File: src/q2g-ext-listboxDirective.ts [hunk 1]
```diff
@@ -0,0 +1,467 @@
...
+//#endregion
+
+//#region
+interface ICreateDim {
+    qGroupFieldDefs: Array<string>;
+    qFallbackTitle: string;
+}
+//#endregion
+
+interface ITest {
```
Comment (konne) [inline src/q2g-ext-listboxDirective.ts]: @thomashaenig whats that

File: src/q2g-ext-listboxDirective.ts [hunk 2]
```diff
@@ -0,0 +1,467 @@
...
+         utils,
+         directives }           from "../node_modules/davinci.js/dist/umd/daVinci";
+import * as template            from "text!./q2g-ext-listboxDirective.html";
+import { RootSingleList, IQlikSingleListController } from "../node_modules/davinci.js/src/utils/rootclasses";
+
+import { ListViewDirectiveFactory } from "../node_modules/davinci.js/src/directives/listview";
+//#endregion
+
+//#region
+interface ICreateDim {
```
Comment (konne) [inline src/q2g-ext-listboxDirective.ts]: @thomashaenig what is here the intention

File: src/q2g-ext-listboxExtension.less [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    .editMode {
+        position: absolute;
+        width: 100%;
+        height: 100%;
+        z-index: 1000;
+        top: 0;
+    }
+}
+
+
```
Comment (konne) [inline src/q2g-ext-listboxExtension.less]: @thomashaenig please remove empty lines, also on other places

File: src/q2g-ext-listboxExtension.less [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+
+
+.q2g-selection {
```
Comment (konne) [inline src/q2g-ext-listboxExtension.less]: @thomashaenig correct namespace?? I think that have to be something with listbox

File: src/q2g-ext-listboxExtension.ts [hunk 5]
```diff
@@ -0,0 +1,246 @@
...
+//#endregion
+
+//#region registrate directives
+var $injector = qvangular.$injector;
+utils.checkDirectiveIsRegistrated($injector, qvangular, "", ListboxDirectiveFactory("Listboxextension"),
+    "ListboxExtension");
+//#endregion
+
+//#region interfaces
+interface IDataProperties {
```
Comment (konne) [inline src/q2g-ext-listboxExtension.ts]: @thomashaenig I see no advantage compared to a direct :any in the show function below, or you can add a :{ properties: any }

**Comment by thomashaenig** [comment]
@konne  new Version

**Comment by konne** [comment]
@thomashaenig please reflect my small requested changes, before I merge the PR

**Pull request merged**